### DISCUSSION TITLE
Running multiple battles in parallel on a server

### DISCUSSION THREAD

**Discussion by yaskovdev** [robocode-dev/tank-royale#57] [Q&A] [answered]
Good evening! Thank you for the new Robocode. :)

I'm trying to use genetic algorithms to evolve a bot. This implies running many battles as fast as possible, so I'm trying to run them in parallel.

Is it possible to use one server instance to run multiple battles at the same time or I have to start as many servers as the parallel battles?

**Comment by flemming-n-larsen** [comment, marked as answer]
@yaskovdev The current version does not support running multiple battles at the same time with the same server. But it sure is possible to run multiple servers at a time as long as you set a new server port for each server, and tell each bot instance which server port to join. 🙂
